tests/phpunit/elementor/includes/container/test-container.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Elementor\Tests\Phpunit\Includes\Container;
+
+use Elementor\Container\Container;
+use Elementor\Core\Isolation\Plugin_Status_Adapter;
+use Elementor\Core\Isolation\Wordpress_Adapter;
+use Elementor\Core\Isolation\Wordpress_Adapter_Interface;
+use Elementor\Core\Utils\Version;
+use Elementor\Plugin;
+use Elementor\Tests\Phpunit\Includes\Container\Traits\Trait_Test_Container;
+use ElementorDeps\DI\DependencyException;
+use ElementorDeps\DI\NotFoundException;
+use ElementorEditorTesting\Elementor_Test_Base;
+use ElementorDeps\DI as DI;
+
+class Test_Container extends Elementor_Test_Base
+{
```

Comment:
```suggestion
class Test_Container extends Elementor_Test_Base {
```
Check the code styling of this file